README.md

```diff
@@ -1473,7 +1473,9 @@ authClient.token.getUserInfo(accessTokenObject)
 
 #### `token.verify(idTokenObject)`
 
-Verify the validity of an ID token's claims and check the signature on browsers that support web cryptography.
+Manually verify the validity of an ID token's claims and check the signature on browsers that support web cryptography.
+
+> **Note:** Token validation occurs [automatically](https://github.com/okta/okta-auth-js/blob/master/lib/token.js#L186-L190) when tokens are returned via `getWithoutPrompt`, `getWithPopup`, and `getWithRedirect`.
```

Comment:
Instead of telling developers that validation occurs automatically in every `token.get*` method, placing it here notifies them before they attempt to use `token.verify`.